### DISCUSSION TITLE
Vehicle Variant Standardization

### DISCUSSION THREAD

**Discussion by RitikaHanish** [ONDC-Official/mobility-specification#112] [General]
@92shreyansh 
There is a requirement to standardize the vehicle category and variants to support seamless inter-operability.

**Comment by NikithShetty** [comment]
We need a reliable way to parse vehicle variant and the service category for all BAPs to render it accordingly to provide UX.
Vehicle variants for `CAB` include `SUV`, `HATCHBACK` and `SEDAN`, similarly for other variants of vehicle types. We need to standardize the minimum set of these variants so that BAP can reliably provide UX.

On top of this, there are service tiers/categories - ex: `Taxi`, `Taxi PRIME` which reflects the service being offered in combination of vehicle variants, amenities, comfort level, etc. This can be dynamic and be represented via item categories.

**Comment by saurabhTriffy** [comment]
The vehicle object has a field 'variant', we can make use of that for all the vehicle variants for CAB like SUV, HATCHBACK, SEDAN etc.
We should also discuss upon which of these will be mandatory and optional, considering if the other party does not has these segregations
